.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Why this matters
+
+<!-- In this section , add details of PR and what changes it brings . Try to explain why this PR matters -->
+
+## Solution
+
+<!-- List all the proposed changes in your PR -->
+
+<!-- Provide breif description of the approach you are using to resolve the issue -->
+
+## How to test it
+
+<!-- This field is optional -->
+
+<!-- Provide the steps to test the changes are working as expected -->
+
+
+## Check List (Check all the applicable boxes)  <!-- Follow the above conventions to check the box -->
+
+- [ ] My code follows the code style of this project.
+- [ ] My change requires changes to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing tests passed.
+- [ ] This PR does not contain plagiarized content.
+- [ ] The title of my pull request is a short description of the requested changes.
+
```

Comment:
Can you remove this section? Let's keep this as simple as possible for now, we can always add more stuff later if think it's needed.
```suggestion
```